kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/ComponentType.java

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.resources;
+
+public enum ComponentType {
+    Kroxylicious
```

Comment:
That's breaking the naming convention - make it all capitals.

Reply:
I was working on latest code smells